secondPart/4_convertObject.js

```diff
@@ -0,0 +1,14 @@
+const removeNullandUndefinedValues = (obj) => {
+    const convertedObj = Object.assign({}, obj)
```

Comment:
use spread instead of ObjectAssign. 
`const convertedObj =  {...obj}`
or new method `structuredClone()`. you can read more about on the internet :)